test/indicators/delete_indicator.js

```diff
@@ -0,0 +1,64 @@
+const test = require('ava');
+
+const server = require('../../server');
+const db = require('../../db');
+const uuid = require('../fixtures/uuid');
+
+test.before(t => {
+  server.register(require('inject-then'), function (err) {
+    if (err) throw err;
+  });
+  return db('indicators').insert({
+    id: uuid.delete,
+    name: 'for deleting',
+    private: true,
+    published: false,
+    owner: 'seed',
+    data: {}
+  });
+});
+
+test('delete an indicator, no token', t => {
+  return server.injectThen({
+    method: 'DELETE',
+    url: `/indicators/${uuid.delete}`
+  }).then((res) => {
+    t.is(res.statusCode, 401, 'Status code is 401');
+  });
+});
+
+test('delete an indicator, user token', t => {
+  return server.injectThen({
+    method: 'DELETE',
+    url: `/indicators/${uuid.delete}`,
+    credentials: {}
```

Comment:
I'm thinking this is potentially confusing. What do you think @drewbo to have a 'private', 'member' or 'reader' role (something to indicate that the user is allowed read access to the private data) in the array? This could clarify some of the logic in a future review of the code.